### PR TITLE
[FLINK-6181][Start scripts] Fix regex in start scripts

### PR DESCRIPTION
**Pull request by rmetzger**
This issue has been reported by a user: http://apache-flink-user-mailing-list-archive.2336050.n4.nabble.com/unable-to-add-more-servers-in-zookeeper-quorum-peers-in-flink-1-2-td12321.html

### CODE REVIEW

File: tools/travis_mvn_watchdog.sh [hunk 1]
```diff
@@ -164,7 +164,7 @@ watchdog () {
 
 # Check the final fat jar for illegal artifacts
 check_shaded_artifacts() {
-	jar tf build-target/lib/flink-dist-*.jar > allClasses
```
Comment (zentol) [inline tools/travis_mvn_watchdog.sh]: What issue is this fixing?

**Comment by greghogan** [comment]
Thanks @rmetzger!

File: tools/travis_mvn_watchdog.sh (hunk 1 above)
Comment (StephanEwen) [inline tools/travis_mvn_watchdog.sh]: I assume with the "-" this only matches something like `flink-dist-1.2-SNAPSHOT` or so and with the fix, it also matched `flink-dist.jar`...

Comment (rmetzger) [inline tools/travis_mvn_watchdog.sh]: The problem is that our dist jar is called `flink-dist_2.10-1.3-SNAPSHOT.jar`. so the old variant didn't match the jar.
This leads to the following error in all travis build:
```
java.io.FileNotFoundException: build-target/lib/flink-dist-*.jar (No such file or directory)
	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:220)
	at java.util.zip.ZipFile.<init>(ZipFile.java:150)
	at java.util.zip.ZipFile.<init>(ZipFile.java:121)
	at sun.tools.jar.Main.list(Main.java:1060)
	at sun.tools.jar.Main.run(Main.java:291)
	at sun.tools.jar.Main.main(Main.java:1233)
``` 
I'm doing `jar tf` here to check if guava and other libraries are not part of the fat jar.

**Comment by zentol** [comment]
merging.

**Pull request closed without merging**